docs/en/connector-v2/sink/Kafka.md

```diff
@@ -93,7 +111,9 @@ sink {
 
 ###  change log
 ####  next version
- 
+
  - Add kafka sink doc 
  - New feature : Kafka specified partition to send 
- - New feature : Determine the partition that kafka send based on the message content
+ - New feature : Determine the partition that kafka send messag based on the message content
+ - New feature : Determine the partition of the kafka send message based on the field name
```

Comment:
Replace with `Configure which field is used as the key of the kafka message better` ?